Plugins/Public/base_plugin/CoreModule.cpp

```diff
@@ -552,6 +577,20 @@ bool CoreModule::Timer(uint time)
 					ConPrint(L"CoreModule::timer space_obj=%u health=%f\n", space_obj, base->base_health);
 
 			}
+			else
```

Comment:
what about `MoneyRepair_IsDisabled`?

Reply:
It would would be sufficient only to add a parameter for
how much relative health base should be for moner repair to start.
for example, if it takes a relative health base more than 1.01, money repairing will never be working for sure.